.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+src/ @woodiertexas @TibiNonEst @JoostMSoftware @peaceheis
```

Comment:
```suggestion
src/main/java/ @woodiertexas @TibiNonEst @JoostMSoftware @peaceheis
```

Reply:
I think that the we can confine that to just Java files, no?

Reply:
that'd work tbh

Reply:
Change implmented.